modules/web/src/app/settings/admin/defaults/static-labels-form/component.ts

```diff
@@ -0,0 +1,159 @@
+// Copyright 2024 The Kubermatic Kubernetes Platform contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Component, EventEmitter, Input, OnDestroy, OnInit, Output} from '@angular/core';
+import {FormArray, FormBuilder, FormGroup, Validators} from '@angular/forms';
+import {ResourceType} from '@app/shared/entity/common';
+import {EMPTY_STATIC_LABELS, StaticLabel} from '@app/shared/entity/settings';
+import {AsyncValidators} from '@app/shared/validators/async.validators';
+import {LabelFormValidators} from '@app/shared/validators/label-form.validators';
+import {Subject, takeUntil} from 'rxjs';
+
+enum Controls {
+  StaticLabels = 'staticLabels',
+  Key = 'key',
+  Values = 'values',
+  Protected = 'protected',
+  Default = 'default',
+}
+
+@Component({
+  selector: 'km-static-labels-form',
+  templateUrl: './template.html',
+  styleUrls: ['style.scss'],
+})
+export class StaticLabelsFormComponent implements OnInit, OnDestroy {
+  private _unsubscribe = new Subject<void>();
+  readonly Controls = Controls;
+
+  form: FormGroup;
+  @Input() staticLabels: StaticLabel[] = [];
+  asyncLabelValidators = [AsyncValidators.RestrictedLabelKeyName(ResourceType.Cluster)];
+  @Output() staticLabelsChange = new EventEmitter<StaticLabel[]>();
+
+  get statickLabelArray(): FormArray {
```

Comment:
Sorry I didn't notice this previously:
```suggestion
  get staticLabelArray(): FormArray {
```